Cannot compile with SQ_DISABLE_INSTALLER

Compiling with `SQ_DISABLE_INSTALLER` activated results in a CMake failure.

Steps to reproduce:
- In a folder `build` inside the repository, run `cmake -DSQ_DISABLE_INSTALLER=ON ..`. This will give the following error:
```
CMake Error at CMakeLists.txt:90 (export):
  export Export set "squirrel" not found.
```

I'm using Linux + CMake 3.16.3.

----------

Note: commenting lines 90 to 93 in CMakeLists.txt fixes the issue; perhaps an `if(NOT SQ_DISABLE_INSTALLER)` is missing?

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -87,12 +87,12 @@ configure_package_config_file(
   INSTALL_DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/squirrel"
   )
 
-export(EXPORT squirrel
-  NAMESPACE squirrel::
-  FILE "${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_INSTALL_LIBDIR}/cmake/squirrel/squirrel-targets.cmake"
-  )
-
 if(NOT SQ_DISABLE_INSTALLER AND NOT SQ_DISABLE_CMAKE_INSTALLER)
+  export(EXPORT squirrel
+    NAMESPACE squirrel::
+    FILE "${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_INSTALL_LIBDIR}/cmake/squirrel/squirrel-targets.cmake"
+    )
+
   install(FILES
     "${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_INSTALL_LIBDIR}/cmake/squirrel/squirrel-config-version.cmake"
     "${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_INSTALL_LIBDIR}/cmake/squirrel/squirrel-config.cmake"

